shell/server/hack-session.js

```diff
@@ -33,6 +33,8 @@ const Grain = Capnp.importSystem("sandstorm/grain.capnp");
 
 const Url = Npm.require("url");
 
+const escapeRegExp = require('lodash.escaperegexp');
```

Comment:
(Or do you have some other way to get NPM packages on the server side. In my Meteor apps I use only packages, never top-level `client`, `server` directories, then it is easy to depend on NPM packages from Meteor packages.)